core/frontend/src/AccuDraw.ts

```diff
@@ -1294,6 +1294,11 @@ export class AccuDraw {
     const rMatrix = out ? out : new Matrix3d();
     rMatrix.setFrom(ViewState.getStandardViewMatrix(nStandard));
     const useVp = vp ? vp : IModelApp.viewManager.selectedView;
+
+    const globeToWorld = vp?.view.getGlobeRotation();

```

Comment:
Probably should only be done when useAcs is false? This method is used by AccuDrawHintBuilder.getContext rotation.

  /** Return a [[Matrix3d]] corresponding to the supplied [[ContextRotationId]].
   * A [[ContextRotationId]] that corresponds to a standard view, "Top", "Front", etc. will return a [[Matrix3d]] that
   * is relative to the current [[Viewport.auxCoordSystem]] when ACS context lock is enabled.
   * @see [[ToolAdmin.acsContextLock]]
   */
  public static getContextRotation(id: ContextRotationId, vp: Viewport): Matrix3d | undefined {

Reply:
Fixed.